### PR TITLE
Update Project Description and Readme

### PR DESCRIPTION
**Pull request by ekigamba**
1. Brief User Documentation
2. Screenshots on all steps
3. List the features of the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,27 +3,110 @@
...
+
+# Features
+
+The app has the following features:
+
+1. It enables one to record the current or past weight
+2. It enables one to edit a weight record
+3. It enables one to view patient weight over time on a graph
+4. It enables one to view the **z-score**
+5. It enables one to determine a patients weight status from the z-score
```
Comment (pld) [inline README.md]: missing possessive apostrophe in patient's

File: README.md [hunk 2]
```diff
@@ -3,27 +3,110 @@
...
+1. When you open the app, the following page is displayed.
+
+![Main Page](https://user-images.githubusercontent.com/31766075/30366688-a33394c6-9874-11e7-9d21-1a408dba867f.png)
+
+ * The **Top Section** has two buttons:
+     - **Record Weight Button** which enables one to record `current` or `past weight`
+
+     ![Record Weight Screenshot](https://user-images.githubusercontent.com/31766075/30361164-1c9a2d64-985e-11e7-8852-099b6d55f577.png)
+     ![Record Past Weight Screenshot](https://user-images.githubusercontent.com/31766075/30361167-1ca12718-985e-11e7-9863-bb4a89efa134.png)
+     - **Weight Graph Button** which displays a dialog with the respective z-scores over the patients lifetime on a graph. The specific values are also displayed below the graph with the patient's age respective age.
```
Comment (pld) [inline README.md]: extra "age" before "respective"

File: README.md [hunk 3]
```diff
@@ -3,27 +3,110 @@
...
+    * Install VirtualBox
+    * Download Genymotion & Install it
+    * Sign in to the genymotion app
+    * Create a new Genymotion Virtual Device 
+        * **Preferrable & Stable Choice** - API 22(Android 5.1.0), Screen size of around 800 X 1280, 1024 MB Memory --> eg. Google Nexus 7, Google Nexus 5
+
+## How to install
+
+1. Import the project into Android Studio by: **Import a gradle project** option
+   _All the plugins required are explicitly stated therefore can work with any Android Studio version - Just enable it to download any packages not available offline_
```
Comment (pld) [inline README.md]: "therefore" => "and therefore it"

Comment (ekigamba) [reply README.md]: 👍

**Pull request merged**